http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20240117_create_app_chart_version_table.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="ladieu" id="create_app_chart_version_table">
+        <createTable tableName="APP_CHART_VERSION">
+            <column name="chartName" type="VARCHAR(254)">
+                <constraints primaryKey="true"/>
```

Comment:
Will this table contain only one version per app? [This section in the design doc](https://broadworkbench.atlassian.net/wiki/spaces/IA/pages/2906914878/Leonardo+App+Versioning+Management#App-vs-Chart-versions) suggests it may contain multiple versions. If so, will `chartName` work as a primary key by itself? Or should `chartVersion` be included in the key?

Reply:
Fixed in 5eefc30f0a772cdd459746bb0218604049e15d2c

Reply:
Ah, I see where the confusion comes from, as the table in the doc is for sherlock and not leonardo. Leo's table should only have one chart version per app name only

Reply:
AKA chartName should be able to work as a PK by itself

Reply:
@Qi77Qi I had a similar question which I discussed with @LizBaldo over VC.  My takeaway was that this is really the name of the _chart_ not the App, and so its relationship with the `APP` model is at most a loose foreign key constraint.  The Chart must be tied to an App with the given name, but it doesn't need to be bound explicitly to a particular `appId`, which would also imply this chart was bound to a particular `nodepoolId`, which is part of the PK of `APP`.

Reply:
oh yea...that makes sense 👍